.github/workflows/review_status.yml

```diff
@@ -0,0 +1,124 @@
+    - name: "Find and update issues in project board"
+      run: |
+        # Now, for each issue we find above, we have some work to do
+        for i in $(jq '.data.repository.pullRequest.closingIssuesReferences.edges[].node.number' ./issues.json)
+        do
+          echo " ISSUE $i "
+          # For each issue, use some graphql to interrogate it to see if it
+          # A) Has an associated issue, and if so
+          # B) If that Issue exists on a project board which has a field
+          # named either "Review 1" or "Review 2".  If it does, 
+          # output the project, field and item id, along with the single select field
+          # optinos for the reviewer submitting this review.  Output that data
+          # to the files review1.json and review2.json
+          gh api -F issue=$i -F reviewer="$REVIEWER" graphql -f query='query($issue: Int!, $reviewer: [String!]) { repository(owner:"openssl", name:"openssl") { issue(number: $issue) { projectItems(first: 50) { edges { node { id project {title id field(name:"Review 1") { ... on ProjectV2SingleSelectField { id options(names: $reviewer) { id color} } } } fieldValueByName(name: "Review 1") { ... on ProjectV2ItemFieldSingleSelectValue { status: name color } } } } } } } }' > ./review1.json
+
+          gh api -F issue=$i -F reviewer="$REVIEWER" graphql -f query='query($issue: Int!, $reviewer: [String!]) { repository(owner:"openssl", name:"openssl") { issue(number: $issue) { projectItems(first: 51) { edges { node { id project {title id field(name:"Review 1") { ... on ProjectV2SingleSelectField { id options(names: $reviewer) { id color} } } } fieldValueByName(name: "Review 1") { ... on ProjectV2ItemFieldSingleSelectValue { status: name color } } } } } } } }' > ./review2.json
+
+          # For each of the two files review1.json and review2.json, do some work
+          for rev in review1.json review2.json
+          do
+            # Fetch the reviewer assigned to this issue in the project board
+            # and extract the PROJECT, FIELD and ITEM id's along with
+            # The single select option that matches the color for the action
+            # Associated with the review
+            ASSIGNED_REVIEWER=$(jq -r '.data.repository.issue.projectItems.edges[0].node.fieldValueByName.status' ./review1.json)
+            PROJECT_ID=$(jq -r '.data.repository.issue.projectItems.edges[0].node.project.id' ./review1.json)
+            FIELD_ID=$(jq -r '.data.repository.issue.projectItems.edges[0].node.project.field.id' ./review1.json)
+            ITEM_ID=$(jq -r '.data.repository.issue.projectItems.edges[0].node.id' ./review1.json)
+            COLOR_OPTION=$(jq -r --arg color "$ACTION_COLOR" '.data.repository.issue.projectItems.edges[0].node.project.field.options[] | select(.color==$color) | .id' ./review1.json)
```

Comment:
Why we did create review1.json and review2.json earlier? Here we are always using review1.json.

Reply:
Thats an error, thank you.  The files there should actually be $rev